### PR TITLE
Docs/review docker readme

### PR DESCRIPTION
**Pull request by frascuchon**
# Description

Closes #<issue_number>

**Type of change**

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- Refactor (change restructuring the codebase without changing functionality)
- Improvement (change adding some improvement to an existing functionality)
- Documentation update

**How Has This Been Tested**

**Checklist**

- I added relevant documentation
- I followed the style guidelines of this project
- I did a self-review of my code
- I made corresponding changes to the documentation
- I confirm My changes generate no new warnings
- I have added tests that prove my fix is effective or that my feature works
- I have added relevant notes to the CHANGELOG.md file (See https://keepachangelog.com/)

### CODE REVIEW

File: argilla-server/docker/argilla-hf-spaces/README.md [hunk 1]
```diff
@@ -13,3 +13,16 @@ Argilla is a **collaboration tool for AI engineers and domain experts** that req
 ## Why use Argilla?
 
 Whether you are working on monitoring and improving complex **generative tasks** involving LLM pipelines with RAG, or you are working on a **predictive task** for things like AB-testing of span- and text-classification models. Our versatile platform helps you ensure **your data work pays off**.
+
+### Environment variables
+
+Apart from the common environment variables defined in docs, the docker image provides a set of variables to simplify the server startup:
```
Comment (dvsrepo) [inline argilla-server/docker/argilla-hf-spaces/README.md]:
```suggestion
Besides the common environment variables defined in docs, this Docker image provides a set of variables to simplify the server startup:
```

File: argilla-server/docker/argilla-hf-spaces/README.md [hunk 2]
```diff
@@ -13,3 +13,16 @@ Argilla is a **collaboration tool for AI engineers and domain experts** that req
...
+
+- `USERNAME`: If provided, the owner username. This can be combined with HF OAuth to define the argilla server owner (Default to `$SPACE_AUTHOR_NAME`).
+
+- `PASSWORD`: If provided, the owner password. If `USERNAME` and `PASSWORD` are provided, the owner user will be created with these credentials on the server startup (Default: `""`).
+
+- `API_KEY`: The default user api key to user. If API_KEY is not provided, a new random api key will be generated (Default: `""`).
```
Comment (dvsrepo) [inline argilla-server/docker/argilla-hf-spaces/README.md]: The API KEY for the owner?

**Review by dvsrepo — APPROVED** [review]
Looks good! Some minor suggestions

File: argilla-server/docker/argilla-hf-spaces/README.md (hunk 2 above)
Comment (frascuchon) [reply argilla-server/docker/argilla-hf-spaces/README.md]: Yes. Before we had DEFAULT_USER_API_KEY

Comment (frascuchon) [inline argilla-server/docker/argilla-hf-spaces/README.md]:
```suggestion
- `API_KEY`: If provided, the owner api key. When `USERNAME` and `PASSWORD` are provided and `API_KEY` is empty, a new random value will be generated (Default: `""`).
```

Comment (frascuchon) [inline argilla-server/docker/argilla-hf-spaces/README.md]:
```suggestion
- `API_KEY`: If provided, the owner API key. If API_KEY is not provided when creating the user, a new random value will be generated (Default: `""`).
```

File: argilla-server/docker/server/README.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+### Environment variables
+
+Apart from the common environment variables defined in docs, the docker image provides a set of variables to simplify the server startup:
+
+- `USERNAME`: If provided, the owner username. This can be combined with HF OAuth to define the argilla server owner (Default: `""`).
+
+- `PASSWORD`: If provided, the owner password. If `USERNAME` and `PASSWORD` are provided, the owner user will be created with these credentials on the server startup (Default: `""`).
+
+- `API_KEY`: The default user api key to user. If API_KEY is not provided, a new random api key will be generated (Default: `""`).
```
Comment (frascuchon) [inline argilla-server/docker/server/README.md]:
```suggestion
- `API_KEY`: If provided, the owner api key. When `USERNAME` and `PASSWORD` are provided and `API_KEY` is empty, a new random value will be generated (Default: `""`).
```

File: argilla-server/docker/server/README.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+Argilla is a **collaboration tool for AI engineers and domain experts** that require **high-quality outputs, data ownership, and overall efficiency**.
+
+## Why use Argilla?
+
+Whether you are working on monitoring and improving complex **generative tasks** involving LLM pipelines with RAG, or you are working on a **predictive task** for things like AB-testing of span- and text-classification models. Our versatile platform helps you ensure **your data work pays off**.
+
+
+### Environment variables
+
+Apart from the common environment variables defined in docs, the docker image provides a set of variables to simplify the server startup:
```
Comment (frascuchon) [inline argilla-server/docker/server/README.md]:
```suggestion
Besides the common environment variables defined in docs, this Docker image provides a set of variables to simplify the server startup:
:
```

**Pull request merged**